rotkehlchen/chain/bitcoin/xpub.py

```diff
@@ -271,18 +262,14 @@ def add_bitcoin_xpub(
                 )
                 self._derive_xpub_addresses(xpub_data, new_xpub=True, blockchain=blockchain)
 
-        if not self.chain_manager.balances.is_queried(blockchain):
-            self.chain_manager.query_balances(blockchain, ignore_cache=True)
-        return self.chain_manager.get_balances_update()
-
     def delete_bitcoin_xpub(
             self,
             write_cursor: 'DBCursor',
             xpub_data: XpubData,
             blockchain: Literal[SupportedBlockchain.BITCOIN, SupportedBlockchain.BITCOIN_CASH],
-    ) -> 'BlockchainBalancesUpdate':
+    ) -> None:
         """
-        Deletes an xpub from the DB, along with all derived addresses, xpub and tag mappings
+        Deletes a xpub from the DB, along with all derived addresses, xpub and tag mappings
```

Comment:
This is correct. Do not introduce typos.

```suggestion
        Deletes an xpub from the DB, along with all derived addresses, xpub and tag mappings
```